src/fidesops/service/connectors/query_config.py

```diff
@@ -256,6 +256,76 @@ def generate_update_stmt(
         returns None"""
 
 
+class ManualQueryConfig(QueryConfig[Executable]):
+    def generate_query(
+        self, input_data: Dict[str, List[Any]], policy: Optional[Policy]
+    ) -> Optional[ManualAction]:
+        """Describe the details needed to manually retrieve data from the
+        current collection.
+
+        Example:
+        {
+            "step": "access",
+            "collection": "manual_dataset:manual_collection",
+            "action_needed": [
+                {
+                    "locators": {'email': "customer-1@example.com"},
+                    "get": ["id", "box_id"]
+                    "update":  {}
+                }
+            ]
+        }
+
+        """
+
+        locators: Dict[str, Any] = self.node.typed_filtered_values(input_data)
+        get: List[str] = [
+            field_path.string_path
+            for field_path in self.node.node.collection.top_level_field_dict
+        ]
+
+        if get and locators:
+            return ManualAction(locators=locators, get=get, update=None)
+        return None
+
+    def query_to_str(self, t: T, input_data: Dict[str, List[Any]]) -> str:
+        """Convert query to string"""
+
+    def dry_run_query(self) -> Optional[str]:
+        """dry run query for display"""
+
+    def generate_update_stmt(
+        self, row: Row, policy: Policy, request: PrivacyRequest
+    ) -> Optional[ManualAction]:
+        """Describe the details needed to manually mask data in the
+        current collection.
+
+        Example:
+         {
+            "step": "erasure",
+            "collection": "manual_dataset:manual_collection",
+            "action_needed": [
+                {
+                    "locators": {'id': 1},
+                    "get": []
+                    "update":  {'authorized_user': None}
+                }
+            ]
+        }
+        """
+        locators: Dict[str, Any] = filter_nonempty_values(
+            {
+                field_path.string_path: field.cast(row[field_path.string_path])
+                for field_path, field in self.primary_key_field_paths.items()
+            }
+        )
+        update_stmt: Dict[str, Any] = self.update_value_map(row, policy, request)
+
+        if update_stmt and locators:
+            return ManualAction(locators=locators, get=None, update=update_stmt)
```

Comment:
Is `get` sure to be `None` or is it silently dropped?

Reply:
get is being saved as None for any `generate_update_stmt` in this design because it's returning an update statement